src/comp_loinc/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .generator import Generator
 from .cli import cli
+
+cli()
```

Comment:
## CLI Questions
1. What is the purpose of this, along with`main.py`, which also has a CLI?
  - I think that `main.py` is the original implementation, and I'm assuming that this is the new one, and is a work in progress, and that we'll delete `main.py` afterwards?
2. Is the class-based approach a standard Typer pattern?
  - It looks a bit Java-esque. I personally find it slightly verbose and confusing (at first glance).